packages/test/test-service-load/src/runner.ts

```diff
@@ -256,6 +256,7 @@ async function runnerProcess(
 			container = await loader.resolve({ url, headers }, stashedOps);
 
 			container.connect();
+			await waitForContainerConnectionWithTimeout(container);
```

Comment:
something like this already happens with  test.run(runConfig, reset). if something is missed there, that would be a better place for it.

also, should we consider not loggering op round trip perf when not connected? as this is a real scenario, and i don't think we would want to log errors when it happens in any case